frontend/src/app/degreeReq/degreeReq.component.ts

```diff
@@ -60,11 +64,20 @@ export class CoursePathComponent implements OnInit {
     })
   }
 
-  getPrereqGraph(){
-    this.api.getData(`${urlConfig.baseUrl}/getPrereqGraph?courseCode=CS%20240&recursionDepth=3`) // HARDCODE TO CS240 FOR NOW!
-    .subscribe(res => {
-      console.log(res);
-    })
+  getPrereqGraph(courseCode: string){
+    this.api.getData(`${urlConfig.baseUrl}/getPrereqGraph?courseCode=${courseCode.trim()}
+        &coursesTaken=${this.coursesTaken}
+        &recursionDepth=3`)
```

Comment:
Can you set the recursion depth to 2 or make the popup scrollable? (Not sure how hard the scrollable part will be to do)

Otherwise, the graph is too tall and gets cut off, like this:
<img width="1000" alt="Screen Shot 2020-04-11 at 12 42 43 PM" src="https://user-images.githubusercontent.com/6634974/79049642-d390b680-7bf2-11ea-93fc-b6f7b0fbf501.png">

Reply:
Made it srollable